code/modules/projectiles/guns/ballistic/shotgun.dm

```diff
@@ -48,6 +50,18 @@
 	sawn_desc = "Come with me if you want to live."
 	can_be_sawn_off  = TRUE
 
+/obj/item/gun/ballistic/shotgun/riot/sawoff(mob/user)
+	. = ..()
+	if(.)
+		weapon_weight = WEAPON_MEDIUM
+		inhand_icon_state = "gun"
+		worn_icon_state = "gun"
+		worn_icon = 'icons/mob/clothing/belt.dmi'
+		lefthand_file = 'icons/mob/inhands/weapons/guns_lefthand.dmi'
+		righthand_file = 'icons/mob/inhands/weapons/guns_righthand.dmi'
```

Comment:
should be handled by base proc.

Reply:
Reverted. Updated the base proc for worn_icon_state. Omitted worn_icon and inhands files.